code/datums/datumvars.dm

```diff
@@ -397,6 +397,7 @@ body
 		html += "[html_encode(name)] = /list ([L.len])"
 
 		if (L.len > 0 && !(name == "underlays" || name == "overlays" || name == "vars" || L.len > 500))
+			name = "[name]"	//Needs to be a string or it will go out of bounds in the internal_byond_list_vars array

```

Comment:
Nope!
According to the runtime logs it passes an integer for a lot of stuff. Mainly lists in lists I think.